src/diagnostics/MOM_diag_to_Z.F90

```diff
@@ -862,11 +862,11 @@ subroutine register_Z_tracer_low(tr_ptr, name, long_name, units, standard_name,
 
   CS%missing_tr(m) = CS%missing_value ! This could be changed later, if desired.
   if (CS%nk_zspace > 0) then
-    CS%id_tr(m) = register_diag_field('ocean_model_z', name, CS%axesTz, Time,           &
+    CS%id_tr(m) = register_diag_field('ocean_model_old_z', name, CS%axesTz, Time,           &
```

Comment:
Let's keep the ocean_model_z intact and use a new module name while we're developing this feature. We'll runn everything through the analysis suite before making the switch.